common-app-lib/test/cmr/common_app/test/services/kms_lookup.clj

```diff
@@ -109,7 +109,13 @@
                        :subregion-2 "CHAD" :subregion-3 "AOUZOU"} "location1-uuid"
 
     "Lookup works without all matching fields present"
-    :spatial-keywords {:category "CONTINENT" :type "AFRICA"} "location2-uuid"))
+    :spatial-keywords {:category "CONTINENT" :type "AFRICA"} "location2-uuid")
+
+  ;; CMR-4400
+  (testing "Platform Shortname exists but Longname is nil"
```

Comment:
We should have four tests total.
1) Long name in KMS, and long name in the metadata - matches.
2) Nil long name in KMS and nil long name in the metadata - matches.
3) Nil long name in KMS, but a long name in the metadata - does not match.
4) Long name in KMS, but a nil long name in the metadata - does not match.

This looks to be test case #4 and it finds a match when it should not (if I'm reading this correctly).

Reply:
4 should match, 3 should not match, 2 should match, and 1 should match.

Reply:
Oh ok I read through the comments in the tickets and the acceptance criteria. I agree 4 should match now.